packages/http_endpoint/_dev/deploy/docker/docker-compose.yml

```
@@ -11,3 +11,14 @@ services:
       - STREAM_USERNAME=abc123
       - STREAM_PASSWORD=abc123
     command: log --start-signal=SIGHUP --webhook-username=abc123 --webhook-password=abc123 --delay=10s /sample_logs/test-webhook.log
+  test-webhook-http-ack:
+    image: docker.elastic.co/observability/stream:v0.16.0
+    volumes:
+      - ./sample_logs:/sample_logs:ro
+    environment:
+      - STREAM_PROTOCOL=webhook
+      - STREAM_ADDR=http://elastic-agent:9516/json?wait_for_completion_timeout=1m
+      - STREAM_WEBHOOK_HEADER=Content-Type=application/json
+      - STREAM_USERNAME=abc123
+      - STREAM_PASSWORD=abc123
+    command: log --start-signal=SIGHUP --webhook-username=abc123 --webhook-password=abc123 --webhook-timeout=2m --delay=10s /sample_logs/test-webhook.log
```

The `STREAM_USERNAME` and `STREAM_PASSWORD` need to be `STREAM_WEBHOOK_{USERNAME,PASSWORD}` (plus they are redundant given the CLI flags). This goes all in on env config:

```suggestion
      - STREAM_WEBHOOK_USERNAME=abc123
      - STREAM_WEBHOOK_PASSWORD=abc123
      - STREAM_WEBHOOK_TIMEOUT=2m
      - STREAM_START_SIGNAL=SIGHUP
      - STREAM_DELAY=10s
    command: log /sample_logs/test-webhook.log
```

---

These reflect what is above. Should I fix those too?